central/processindicator/datastore/datastore_impl.go

```diff
@@ -106,7 +106,9 @@ func (ds *datastoreImpl) AddProcessIndicators(ctx context.Context, indicators ..
 	if err != nil {
 		return err
 	}
-
+	if env.PostgresDatastoreEnabled.BooleanSetting() {
```

Comment:
nit:  comment here might be useful.

Reply:
I just removed this part now that I understand that the baseline stuff is probably the majority